substra/sdk/client.py

```diff
@@ -116,19 +106,37 @@ def from_config_file(
         cls,
         profile_name: typing.Optional[str] = None,
         config_path: typing.Union[str, pathlib.Path, None] = None,
-        user_path: typing.Union[str, pathlib.Path, None] = None,
+        tokens_path: typing.Union[str, pathlib.Path, None] = None,
         token: typing.Optional[str] = None,
         retry_timeout: int = DEFAULT_RETRY_TIMEOUT,
-        backend: str = 'remote',
     ):
-        cfg_manager = cfg.Manager(config_path)
-        profile = cfg_manager.from_config_file(name=profile_name)
+        """Returns a new Client configured with profile data from configuration files.
```

Comment:
Maybe mention that it is only for a remote backend ?

Reply:
Absolutely!